### PR TITLE
Changes for adding mixtral to gh action self-hosted

### PR DESCRIPTION
**Pull request by anandhu-eng**

### CODE REVIEW

File: script/app-mlperf-inference-mlcommons-python/_cm.yaml [hunk 1]
```diff
@@ -1018,6 +1018,7 @@ variations:
     group: models
     env:
       CM_MODEL: mixtral-8x7b
+      CM_ML_MODEL_STARTING_WEIGHTS_FILENAME: "https://github.com/anandhu-eng/cm4mlops/blob/5f156eb891b86a4ea7f1864075ddf1f1ade66598/script/get-ml-model-mixtral/_cm.json#L66"
```
Comment (arjunsuresh) [inline script/app-mlperf-inference-mlcommons-python/_cm.yaml:1021]: We should add this from the mlcommons repo

File: script/get-mlperf-inference-src/customize.py [hunk 2]
```diff
@@ -45,7 +45,7 @@ def preprocess(i):
             env["CM_GIT_URL"] = "https://github.com/mlcommons/inference"
     
     if env.get("CM_MLPERF_LAST_RELEASE", '') == '':
-        env["CM_MLPERF_LAST_RELEASE"] = "v4.0"
+        env["CM_MLPERF_LAST_RELEASE"] = "v4.1"
```
Comment (arjunsuresh) [inline script/get-mlperf-inference-src/customize.py:48]: When we use `v4.1-dev` variation (the default) this should be set to v4.0 from the meta.

File: script/app-mlperf-inference-mlcommons-python/_cm.yaml (hunk 1 above)
Comment (anandhu-eng) [reply script/app-mlperf-inference-mlcommons-python/_cm.yaml]: Updated: https://github.com/mlcommons/cm4mlops/pull/353/commits/7163f9b765de01301b624c536f1d4c8a24c16f98

File: script/get-mlperf-inference-src/customize.py (hunk 2 above)
Comment (anandhu-eng) [reply script/get-mlperf-inference-src/customize.py:48]: Updated: [624cadf](https://github.com/mlcommons/cm4mlops/pull/353/commits/624cadfba8089856ea2815854cb33b7bf157aa5b)

**Pull request closed without merging**